### PR TITLE
Fix crash for top level implicit class declaration

### PR DESCRIPTION
**Pull request by Varunram**
Intended as a fix for #2463

### CODE REVIEW

**Comment by nicolasstucki** [comment]
That line looks a bit too long, could you shorten it like this:
```scala
val exampleArgs = 
  if(constr0.vparamss.isEmpty) "..." 
  else constr0.vparamss(0).map(_.withMods(untpd.Modifiers()).show).mkString(", ")
```

**Comment by nicolasstucki** [comment]
You should also add the test file and rename the commit to `Fix #2463: Fix crash for top level implicit class declaration` (this way github links the commit with the issue).

Thanks for the PR

**Comment by Varunram** [comment]
Your wish is my command! XD

**Comment by Varunram** [comment]
Sorry, just saw the second half of your message. Tests? I thought they were the same, aren't they?

**Comment by nicolasstucki** [comment]
Yes, I meant the `tests/neg/i2463.scala` file.

**Comment by Varunram** [comment]
Done! The reason I asked you was I couldn't find any other test file that needed changes. Thanks a lot!

File: tests/neg/i2463.scala [hunk 1]
```diff
@@ -0,0 +1 @@
+implicit class Foo //error
```
Comment (nicolasstucki) [inline tests/neg/i2463.scala]: You need to add a space between `//` and `error`. The framework looks for the substring `// error` in the source.

Comment (nicolasstucki) [inline tests/neg/i2463.scala]: I fixed it directly from my browser, this should work now.

Comment (Varunram) [inline tests/neg/i2463.scala]: Oh no, I too pushed simultaneously, correcting it now

**Comment by Varunram** [comment]
I force-pushed to fix the commit history. Sorry!

File: tests/neg/i2463.scala (hunk 1 above)
Comment (nicolasstucki) [inline tests/neg/i2463.scala]: Your fast

**Pull request merged**